packages/ui-extensions-react/src/surfaces/checkout/render.tsx

```diff
@@ -17,7 +17,7 @@ import {ExtensionApiContext} from './context';
  * hook.
  *
  * @param target The extension target you are registering for. You can see a full list
- * of the available targets in our [developer documentation](https://shopify.dev/docs/api/checkout-ui-extensions/extension-targets-overview#placements).
+ * of the available targets in our [developer documentation](https://shopify.dev/docs/api/checkout-ui-extensions/extension-targets-overview#supported-locations).
```

Comment:
Seems like it's been [changed some time ago by Chris](https://github.com/Shopify/checkout-web/pull/23800/files), but never made it to a release (unless I'm misunderstanding the flow). Can we keep it as part of this release @jamesvidler?

Note: after reviewing the whole code diff, it seems like other changes below are related.

Reply:
Yes, these content changes are fine to release as part of this. Thanks for checking!